modules/ensemble/lib/framework/apiproviders/firestore/firestore_api_provider.dart

```diff
@@ -58,7 +58,7 @@ class FirestoreAPIProvider extends APIProvider with LiveAPIProvider {
     } else {
       // Initialize the new Firebase app with the options
       FirebaseApp app = await Firebase.initializeApp(
-        name: appId,
+        name: 'customFirebase',
```

Comment:
huh?

Reply:
firebase auth initialisation:

https://github.com/EnsembleUI/ensemble/blob/3b31f77b6791bfe761699a8b6408cc32967553c6/modules/auth/lib/signin/auth_manager.dart#L178-L179

Reply:
I understand, my question was why hardcode the name of the app? is it supposed to match something somewhere else? where?

Reply:
we just need to name firebase app, since we have ensemble firebase app initialised. 
Only the firebase auth and firestore need to match the name of firebase app. 

Either way we'll need to have single initialisation of firebase service to not worry about matching everywhere.

Reply:
the reason I did it this way was to allow multiple apps to run in Ensemble Go and the Studio. That use case will fail if we hardcode the app name right?

Reply:
In that case, we can keep firestore initialisation as it is and update the auth side.

Reply:
yes - on the auth side we can just use the appId as well and then the two will be the same right? I am not sure if that'll be an issue or not?